src/OSPSuite.Presentation/Presenters/ContextMenus/CommonMenuBarButtons.cs

```diff
@@ -62,5 +60,14 @@ public static IMenuBarItem JournalDiagramView(MenuBarItemId menuBarItemId)
             .WithCommand<JournalDiagramVisibilityUICommand>()
             .WithIcon(ApplicationIcons.JournalDiagram);
       }
+
+      public static IMenuBarItem ClearHistory(MenuBarItemId menuBarItemId)
```

Comment:
I created a button that can be used in ribbon as well as an entry in history presente.r That way we have the best of both worlds

Reply:
>... the best of both worlds

"Best of all worlds" always reminds me about [this](http://raybradbury.ru/library/story/60/4/0/)  :wink: